src/main/resources/application.yaml

```diff
@@ -112,7 +112,7 @@ feign:
   allow:
     headers:
       - authorization
-userProfUrl: ${USER_PROFILE_URL:http://rd-user-profile-api-aat.service.core-compute-aat.internal}
+userProfUrl: http://rd-user-profile-api-pr-664.service.core-compute-preview.internal
```

Comment:
The changes made here and in `application-functional.yaml` to UP URL will be **reverted** once PR has been approved and QA has been completed successfully.

Reply:
Blocking the merge until this being done